java/bench/core/src/java/org/apache/orc/bench/core/IOCounters.java

```diff
@@ -48,23 +50,26 @@ public void print() {
     if (recordCounters != null) {
       recordCounters.print();
     }
-    System.out.println("Reads: " + reads);
-    System.out.println("Bytes: " + bytesRead);
+    System.out.println("io: " + io);
+    System.out.println("Bytes: " + bytesIO);
   }
 
   public double bytesPerRecord() {
     return recordCounters == null || recordCounters.records == 0 ?
-        0 : ((double) bytesRead) / recordCounters.records;
+        0 : ((double) bytesIO) / recordCounters.records;
   }
 
   public long records() {
     return recordCounters == null || recordCounters.invocations == 0 ?
         0 : recordCounters.records / recordCounters.invocations;
   }
 
-  public long reads() {
+  /**
+   * Capture the number of I/O on average in each invocation.
+   */
+  public long iOs() {
```

Comment:
IOps?

Reply:
thanks that's a much better name !

Reply:
Second thoughts: there's no "per second" so I may just called it `ops`, wdyt ?